components/partials/portfolio/holdings/index.vue

```diff
@@ -86,14 +88,43 @@ export default Vue.extend({
             (token) => token.denom.toLowerCase() === balance.denom.toLowerCase()
           )
 
+          const balanceInUsd = tokenWithBalance
+            ? new BigNumberInWei(balance.totalBalance)
+                .toBase(tokenWithBalance.decimals)
+                .times(tokenWithBalance?.priceInUsd || 0)
+            : new BigNumberInWei(0)
+
           return {
             ...balance,
+            balanceInUsd,
             token: tokenWithBalance
           }
         })
         .filter(
           (balance) => !!balance.token
-        ) as SubaccountBalanceWithTokenMetaData[]
+        ) as SubaccountBalanceWithTokenMetaDataWithUsdBalance[]
+    },
+
+    sortedBalances(): SubaccountBalanceWithTokenMetaDataWithUsdBalance[] {
+      const { balances } = this
+
+      return balances.sort(
```

Comment:
Let's copy the array before sorting it because `.sort` is applied on the original array. `[...balances].sort(`

You don't need to do this in this file: `components/partials/wallet/balances/index.vue` i.e below because `.map` returns a new array and doesn't alter the original one.

Reply:
ah, you've mentioned this before. Thanks for sharing, I've just pushed the fix for this.